### PR TITLE
feat: llama-vision-audio model

### PR DESCRIPTION
**Pull request by XuyaoWang**
# Description

Describe your changes in detail.

## Motivation and Context

Why is this change required? What problem does it solve?
If it fixes an open issue, please link to the issue here.
You can use the syntax `close #1314520` if this solves the issue #15213

- [ ] I have raised an issue to propose this change ([required](https://github.com/PKU-Alignment/align-anything/issues) for new features and bug fixes)

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds core functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [ ] I have read the [CONTRIBUTION](https://github.com/PKU-Alignment/align-anything/blob/HEAD/.github/CONTRIBUTING.md) guide. (**required**)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly. (*required for a bug fix or a new feature*)
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by cby-pku** [comment]
Woaw! Great work!!

**Review by cby-pku — APPROVED** [review]
LGTM

File: align_anything/configs/template.py [hunk 1]
```diff
@@ -114,16 +152,257 @@ def format_prompt_only_sample(self, raw_sample: dict[str, Any]) -> dict[str, Any
...
+            'image': Image.open(image_file),
+        }
+
+    
 @register_template('LLAVA')
 class LLAVA:
     system_prompt: str = ''
     user_prompt: str = 'USER: \n<image>{input}'
-    assistant_prompt: str = '\nASSISTANT:{output}'
+    assistant_prompt: str = '\nASSISTANT: {output}'
```
Comment (Gaiejj) [inline align_anything/configs/template.py:401]: why space here

File: align_anything/configs/template.py [hunk 2]
```diff
@@ -147,6 +426,114 @@ def format_sample(self, raw_sample: dict[str, Any]) -> dict[str, Any]:
...
+        prompt = (
+            f'{self.user_prompt.format(input=question)}'
+            f"{self.assistant_prompt.format(output='')}"
+        )
+        audio = raw_sample['audio']['array']
+        return {
+            'text': text,
+            'prompt': prompt,
+            'audio': audio,
+            'sample_rate': 48000
```
Comment (Gaiejj) [inline align_anything/configs/template.py:507]: the sample_rate seems fixed, is it right?

File: align_anything/configs/template.py (hunk 1 above)
Comment (XuyaoWang) [reply align_anything/configs/template.py]: has fixed

File: align_anything/configs/template.py (hunk 2 above)
Comment (XuyaoWang) [reply align_anything/configs/template.py]: has fixed

**Review by Gaiejj — APPROVED** [review]
LGTM

**Pull request merged**